Knossos.NET/Views/Windows/PerformanceHelpView.axaml

```diff
@@ -12,8 +12,8 @@
 		CanResize="False">
 	
 	<StackPanel MaxWidth="500" Background="{StaticResource BackgroundColorPrimary}">
-		<TextBlock Margin="10,20,20,10" TextWrapping="Wrap">If in-game performance is slow, close the game and then try disabling "Extra Multi Sample Anti-Aliasing" and "Shadows".</TextBlock>
```

Comment:
I think this is a good way to do it.  My inclination is to mention here that the other AA will still work with MSAA off and to add to the msaa tooltip, mentioning exactly what it's for.

Reply:
Thanks, pushed an update with a host of video related tool tip improvements.